chain/network.go

```diff
@@ -86,7 +86,7 @@ func Mainnet() (*consensus.Network, types.Block) {
 		}{
 			AllowHeight:    526000, // June 6th, 2025 @ 6:00am UTC
 			RequireHeight:  530000, // July 4th, 2025 @ 2:00am UTC
-			FinalCutHeight: 600000,
+			FinalCutHeight: 552100, // December 1st, 2025 @ 11:00am UTC
```

Comment:
Please double check the math. I did round it down since it won't be accurate anyway but it should be in the ballpark.

Reply:
```
dec_1 - aug_15 = 108 days
current_height = 536570
(108 * 144) + 536570 = 552122 
```
alternative check:
```
552100 - 530000 = 22100
22100 * 10 minutes = 153 days
jul_4 + 153 days = dec_4
```
lgtm. A 3 day spread is obviously not ideal, but it's tolerable.